shark_turbine/kernel/wave/utils.py

```diff
@@ -375,6 +400,8 @@ def compile_and_invoke(
             **benchmark_flags,
         )
         _print_bench_result(benchmark_results, bench_file)
+        for tf in tempfiles:
```

Comment:
The files will closed when `tempfiles` list is destroyed (or or can `del tempfiles` manually), don't need to call `close` on each file manually.

Reply:
Do you mean when we have "delete=True", we do not need to close the file?